cypress/e2e/autoLogin.cy.ts

```diff
@@ -0,0 +1,221 @@
+import { API_ROUTES } from '@graasp/query-client';
+import {
+  CompleteGuest,
+  DiscriminatedItem,
+  FolderItemFactory,
+  GuestFactory,
+  HttpMethod,
+  ItemLoginSchemaFactory,
+  ItemLoginSchemaType,
+} from '@graasp/sdk';
+
+import { StatusCodes } from 'http-status-codes';
+
+import { buildAutoLoginPath, buildContentPagePath } from '@/config/paths';
+import {
+  AUTO_LOGIN_CONTAINER_ID,
+  AUTO_LOGIN_ERROR_CONTAINER_ID,
+  AUTO_LOGIN_NO_ITEM_LOGIN_ERROR_ID,
+} from '@/config/selectors';
+
+import { API_HOST, AUTH_HOST } from '../support/env';
+import { expectFolderLayout } from '../support/integrationUtils';
+
+const {
+  buildPostItemLoginSignInRoute,
+  buildGetItemLoginSchemaTypeRoute,
+  buildGetCurrentMemberRoute,
+  buildGetItemRoute,
+} = API_ROUTES;
+
+class TestHelper {
```

Comment:
Interesting, the goal is to have a different setup than the current `server`?

Reply:
This is to have a "reactive" server that changes the responses based on the calls you make. So for example it start with logged out user and if you call postItemLogin, then you are logged in, so current member is now returned. This server is dynamic, where the current server is static and always returns the same data in a test.